src/app/layout.tsx

```diff
@@ -1,9 +1,11 @@
+"use client";
+
 import { RootStyleRegistry } from "@/components/global/RootStyleRegistry";
 import type { Metadata } from "next";
 import { GoogleAnalytics } from "@next/third-parties/google";
 import "./globals.css";
 
-export const metadata: Metadata = {
+const metadata: Metadata = {
```

Comment:
is this being used wtihout the export? idts

Reply:
it's not but is it okay to remove?